src/test/java/io/seqera/tower/cli/runs/RunsCmdTest.java

```diff
@@ -553,4 +554,65 @@ void testDownloadTaskLog(MockServerClient mock) throws IOException {
         assertEquals(new RunFileDownloaded(file, RunDownloadFileType.stdout).toString(), out.stdOut);
         assertEquals(0, out.exitCode);
     }
+
+    @Test
+    void testDumpRuns(MockServerClient mock) throws IOException {
+
+        mock.when(
```

Comment:
Same for the test, I think it would be nice to abstract the duplicated code into a single method.

Reply:
Do you mean setting up the test endpoints in a separate method?